eclair-core/src/main/scala/fr/acinq/eclair/Eclair.scala

```diff
@@ -69,15 +70,15 @@ trait Eclair {
 
   def open(nodeId: PublicKey, fundingAmount: Satoshi, pushAmount_opt: Option[MilliSatoshi], fundingFeerateSatByte_opt: Option[Long], flags_opt: Option[Int], openTimeout_opt: Option[Timeout])(implicit timeout: Timeout): Future[ChannelCommandResponse]
 
-  def close(channelIdentifier: Either[ByteVector32, ShortChannelId], scriptPubKey_opt: Option[ByteVector])(implicit timeout: Timeout): Future[ChannelCommandResponse]
+  def close(channels: List[Either[ByteVector32, ShortChannelId]], scriptPubKey_opt: Option[ByteVector])(implicit timeout: Timeout): Future[Map[Either[ByteVector32, ShortChannelId], ChannelCommandResponse]]
```

Comment:
nit: we could reduce the verbosity of the code here and below by introducing a type alias like:
```
  type ChannelIdentifier = Either[ByteVector32, ShortChannelId]
```
and then replacing all instances of `Either[ByteVector32, ShortChannelId]` with `ChannelIdentifier`

Reply:
I went back and forth on that one because I didn't know where to put it...
I wasn't a big fan of putting this directly in the `fr.acinq.eclair` namespace, and couldn't find the right object to put it in...let me try again.

Reply:
Done in https://github.com/ACINQ/eclair/pull/1440/commits/75fd29f9229c6c3666c3fac11df59f5225dc1907